### ISSUE TITLE
Confusing icons for auto-reload and refresh

### ISSUE THREAD

**Issue by tormodvm** [porsager/flems#71]
I've found that my students are having a hard time wrapping their heads around these icons and what they do.  When they want something to run, they press play, and then they don't understand why that button has suddenly disappeared.

I think it's especially confusing for the ones with experience from Scratch.

Could this be done in a different way? There's no other obvious icon that comes to mind, though.
Maybe a text that says "auto-reload" with a checkbox next to it?

(btw. Have I reported this before? Had an ugly feeling of deja vu while typing. :D in that case, sorry...)

**Comment by osban** [comment]
@tormodvm 😀 I would've thought that Play/Pause and Refresh buttons were pretty clear, and there are also the tooltips?

**Comment by tormodvm** [comment]
Yeah, I would've thought that as well, but there's been some confusion. It's hardest for the ones coming from Scratch, where the play button means the same as the refresh button in flems.

**Comment by osban** [comment]
I can imagine, but isn't that more a Scratch issue than a Flems one? Esp. the refresh button is consistent with the refresh button in browsers.

**Comment by tormodvm** [comment]
The refresh button can stay, but I'm not sure if a play icon is the best icon for auto-reload.

**Comment by porsager** [comment]
Yes, the play/pause icon is not optimal. I also find it a bit weird in the top right corner as it feels more like a part of the editor. So perhaps an auto-reload checkbox inside the editor in the lower right corner?

**Comment by tormodvm** [comment]
A checkbox would be nice!
I think the placement is okay as is, but that's up to you. :)

**Comment by porsager** [comment]
By the way, if you don't need the play / pause you can hide it.. Would that make things better for now?

**Comment by tormodvm** [comment]
Yeah, I've done that sometimes, but this was more of a visual enhancement for the times when I actually need it.

**Comment by porsager** [comment]
I think it'd be a pity to clutter with  `auto-reload [ checkbox ]` so any other ideas are more than welcome.

**Comment by pygy** [comment]
It took me a while to remember which was which as wel...

What about this (glowing/dim depending on state)? "Live" may clash with UI related to streaming but hopefully this prods ideas...

![external-content duckduckgo com](https://user-images.githubusercontent.com/54515/80259430-d88b4680-8685-11ea-9ad7-8874c49d5602.jpg)
